app/assets/javascripts/app-level.js

```diff
@@ -1,25 +1,27 @@
 /* JavaScript code that will be added to every page throughout the application */
 
 
-/* Adding CSRF token to every Ajax call by default */
+/* Moment JS generic converter */
+$(document).ready(function() {
+	$(".moment-date-time").each(function(ind) {
+		$el = $(this);
+		var format = $el.data("format")  || "MMMM Do YYYY, h:mm:ss a";
+		var unformattedDate = $el.html();
+		var formattedDate = moment(unformattedDate, "YYYY-MM-DD hh:mm:ss ZZ").format(format);
+		$el.html(formattedDate);
+	});
 
-jQuery.ajaxSetup({
-    headers: {
-        'X-CSRF-Token' : $('meta[name="csrf-token"]').attr('content')
-    },
-});
+	/* Adding CSRF token to every Ajax call by default */
+	jQuery.ajaxSetup({
+    	headers: {
+        	'X-CSRF-Token' : $('meta[name="csrf-token"]').attr('content')
+    	},
+	});
 
+	/* Activating Tooltips */
+	$(function () {
+  		$('[data-toggle="tooltip"]').tooltip()
```

Comment:
I know you didn't add this, but I just noticed this tooltip shows up as a blank square for me when hovering over attachments. This does seem to be bootstrap thing? Perhaps we should just replace/remove it.